### DISCUSSION TITLE
Preview without domain

### DISCUSSION THREAD

**Discussion by Blinks44** [Dokploy/dokploy#77] [General]
Hi! Is it possible to preview deployed project without buying a domain? Just like coolify does internal url for every project?

**Comment by Mr1Blaze** [comment]
Exist free dns like duckdns you can try with this. It's working. But it's working via ip:port

**Comment by Siumauricio** [comment]
It may be a good idea, but in terms of security, performance, privacy, can be slow among other things I would not recommend using it.

A better idea would be to point an * A record to the vps, this way we could create a way to generate these domains automatically, something like what this comment suggests sounds much more robust.

https://github.com/Dokploy/dokploy/issues/40#issuecomment-2094198606

Probably not a very easy option for everyone, but you can buy domains for 1$ and it would be useful for any kind of testing.

**Comment by Siumauricio** [comment]
@Blinks44 Feature implemented, now you can preview projects without having a domain
